homeassistant/components/vesync/sensor.py

```diff
@@ -49,14 +51,17 @@
 class VeSyncSensorEntityDescription(SensorEntityDescription):
     """Describe VeSync sensor entity."""
 
-    value_fn: Callable[[VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch], StateType]
+    value_fn: Callable[
+        [VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch | VeSyncHumidifierDevice],
+        StateType,
+    ]
 
-    exists_fn: Callable[[VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch], bool] = (
-        lambda _: True
-    )
-    update_fn: Callable[[VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch], None] = (
-        lambda _: None
-    )
+    exists_fn: Callable[
+        [VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch | VeSyncHumidifierDevice], bool
+    ] = lambda _: True
+    update_fn: Callable[
+        [VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch | VeSyncHumidifierDevice], None
```

Comment:
I am thinking we should figure out how to manage this.  Seeing a very different approach here than others.  Could we chat more on this on discord so all our PRs are going the same direction?

Reply:
I reduced this to just use `VeSyncBaseDevice` instead of `VeSyncHumidifierDevice`. I have an idea to use generic which I will submit separately since it requires non-humidifier specific updates.